react/components/CustomTags.tsx

```diff
@@ -1,111 +1,148 @@
-  <p className="t-body lh-title">{children}</p>
-)
+export const CustomRenderers = {
+  break: () => <br />,
+  code: (props: any) => {
+    const codeBlock = props.value.replace(/“/gm, '"').replace(/”/gm, '"')
 
-const customH1: FunctionComponent = ({ children }) => (
-  <h1
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-2 bb b--muted-4">
-    {children}
-  </h1>
-)
-
-const customH2: FunctionComponent = ({ children }) => (
-  <h2
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-3">
-    {children}
-  </h2>
-)
-
-const customH3: FunctionComponent = ({ children }) => (
-  <h3
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-4">
-    {children}
-  </h3>
-)
-
-const customH4: FunctionComponent = ({ children }) => (
-  <h4
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-5">
-    {children}
-  </h4>
-)
-
-const customH5: FunctionComponent = ({ children }) => (
-  <h5
-    id={`${children}`.toLowerCase().replace(' ', '-')}
-    className="t-heading-6">
-    {children}
-  </h5>
-)
-
-const customTableHeader: FunctionComponent = ({ children }) => (
-  <th className="tc ph4">{children}</th>
-)
-
-const customTableData: FunctionComponent = ({ children }) => (
-  <td className="tc pv4 ph4">{children}</td>
-)
-
-const customPre: FunctionComponent = ({ children }) => (
-  <pre>
-    <SyntaxHighlighter language="javascript" style={coy}>
-      {children}
-    </SyntaxHighlighter>
-  </pre>
-)
-
-const customListItem: FunctionComponent = ({ children }) => (
-  <li>
-    <p className="t-body lh-title">{children}</p>
-  </li>
-)
-
-const customAnchor: FunctionComponent<AnchorHTMLAttributes<any>> = ({
-  href,
-  children,
-}) => {
-  /* eslint-disable react-hooks/rules-of-hooks */
-  const { route } = useRuntime()
-
-  const isIdLink = !!href && href[0] === '#'
-  const isRelativeLink = !!href && href[0] === '/'
-
-  if (isIdLink && !!href) {
     return (
-      <AnchorLink offset={() => 80} href={href.toLowerCase()}>
-        {children}
-      </AnchorLink>
+      <div className="overflow-scroll-gradient">
+        <pre className="w-100 overflow-y-auto overflow-scroll-gradient__scroller">
+          <SyntaxHighlighter language="javascript" style={coy}>
+            {codeBlock}
+          </SyntaxHighlighter>
+        </pre>
+      </div>
     )
-  }
-
-  if (isRelativeLink) {
-    return <a href={`/docs/${route.params.app}${href}`}>{children}</a>
-  }
+  },
+  emphasis: (props: any) => <em className="i">{props.children}</em>,
+  heading: (props: any) => {
+    const hashId = `${props.children}`.toLowerCase().replace(' ', '-')
+
+    switch (props.level) {
+      case 1:
+        return (
+          <h1 id={hashId} className="t-heading-1 c-on-base mt7 mb7">
+            {props.children}
+          </h1>
+        )
+
+      case 2:
+        return (
+          <h2 id={hashId} className="mt9 mb5 t-heading-2 c-on-base">
```

Comment:
⛏ I would keep all of those `t-heading-2` at the beginning of the classes

Reply:
Basically keep the same order

Reply:
Makes sense!